frontends/main/src/page-components/Header/Header.tsx

```diff
@@ -272,11 +288,24 @@ const Header: FunctionComponent = () => {
             <MenuButton
               ref={desktopTrigger}
               text="Explore MIT"
-              onClick={toggleDrawer.toggle}
+              onClick={() => {
+                toggleDrawer.toggle()
+                if (process.env.NEXT_PUBLIC_POSTHOG_API_KEY) {
+                  posthog.capture(drawerToggleEvent)
+                }
+              }}
             />
           </DesktopOnly>
           <MobileOnly>
-            <MenuButton ref={mobileTrigger} onClick={toggleDrawer.toggle} />
+            <MenuButton
+              ref={mobileTrigger}
+              onClick={() => {
+                toggleDrawer.toggle()
```

Comment:
minor nit - this code block appears to be duplicated in a few places and could be consolidated